network-api/networkapi/urls.py

```diff
@@ -27,7 +27,11 @@
 
 urlpatterns = list(filter(None, [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^soc/', include('social_django.urls', namespace='social'))
+    url(r'^soc/', include('social_django.urls', namespace='social')),
+    # Don't remove this redirect until Fellowships pages are live
+    url(r'^fellowships/$', RedirectView.as_view(
+        url='https://advocacy.mozilla.org/open-web-fellows'
+        ))
```

Comment:
fix indentation, and add a trailing comma, please!

Reply:
Fixed indent, trailing comma results in a syntax error. Notice it wasn't there on the removed line.

Reply:
oh, that's interesting!